docs/getting-started/azure-to-cluster/signup.rst

```diff
@@ -0,0 +1,74 @@
+.. _signup-azure-to-cluster:
+
+===================================================
+Sign up for CrateDB Cloud via the Azure Marketplace
+===================================================
+
+Currently the only way to deploy a CrateDB cluster on CrateDB Cloud is via the
+Azure Marketplace. You will therefore need a Microsoft Azure account and Azure
+subscription with a credit card linked to it. By using CrateDB Cloud's service
+on Azure Marketplace, your hourly usage is billed directly by Microsoft rather
+than by Crate.io.
+
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _signup-azure-to-cluster-offer:
+
+Using the CrateDB Cloud offer on Azure Marketplace
+==================================================
+
+Visit the CrateDB Cloud offer on Azure Marketplace via `this link`_. Under the
+*Plans + Pricing* tab you will find useful information on available cluster
+configurations. The Development plan is suitable as a demo of CrateDB Cloud.
+For production ready use cases, there is a General Purpose plan in either Basic
+or Pro tier, depending on your ingest and query requirements.
+
+Once you know which plan suits your needs, click the blue *Get It Now* button
+to the left to avail yourself of the CrateDB Cloud offer. After giving consent
```

Comment:
"avail yourself of" is potentially confusing. any way to reword to more simple english?